plugins/GranularPitchShifter/GranularPitchShifterEffect.cpp

```diff
@@ -275,12 +275,12 @@ void GranularPitchShifterEffect::changeSampleRate()
 	
 	m_dcCoeff = std::exp(-2.0 * F_PI * DcRemovalHz / m_sampleRate);
 
-	const float pitch = m_granularpitchshifterControls.m_pitchModel.value();
-	const float pitchSpread = m_granularpitchshifterControls.m_pitchSpreadModel.value() * 0.5f;
+	const float pitch = m_granularpitchshifterControls.m_pitchModel.value() * (1. / 12.);
```

Comment:
```suggestion
    const float pitch = m_granularpitchshifterControls.m_pitchModel.value() * (1.f / 12.f);
```

Reply:
ignore the spacing, i typed in 4 spaces because i couldn't type tab in the web editor